doc/blackbird_python/installing.rst

```diff
@@ -4,6 +4,9 @@ Installation
 ============
 
 
+.. note:: Update once uploaded to the PyPI repository
```

Comment:
I think we should make a habit of putting these kinds of things elsewhere (not in the docs). There is too much potential for us to forget that they are there.